fenix/app/src/test/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckStoreTest.kt

```diff
@@ -1282,6 +1282,55 @@ class ReviewQualityCheckStoreTest {
             assertTrue(observedState.contains(expectedProgressState3))
         }
 
+    @Test
+    fun `GIVEN analysis succeeds WHEN deletedProduct is true and deletedProductReported is false THEN product not available is shown`() =
```

Comment:
GIVEN deletedProduct is true and deletedProductReported is false WHEN report product back in stock is clicked THEN thanks for reporting card is shown